### ISSUE TITLE
[tx] UFO glif parsing fails on contours with a single point

### ISSUE THREAD

**Issue by punchcutter** [adobe-type-tools/afdko#1540]
Parsing shouldn't just fail here. In this case the single points shouldn't be there, but tx could issue a warning and continue instead of failing with `tx: (ufr) parsing failed` and no other information.

**Comment by kaydeearts** [comment]
@hatchzo also added:
> The zigzag/loop paths should be corrected by design and tx should report detailed information on parse errors.
But I think the most serious problem is checkoutlineufo. It seems that those single point contours are generated while running checkoutlineufo.
I made a test ufo file that only collect glyphs that cause errors and then run that :
`checkoutlinesufo -e  -w -o test-check.ufo test.ufo`

**Comment by skef** [comment]
@punchcutter it looks like #1786 does address #1632 but this issue also asks to change the error to a warning. Is that no longer desired/needed?

**Comment by punchcutter** [comment]
I don't think we need anything else. If we get this new better message telling us which glyph had a problem then the user can check that glyph and try again.
